mlflow/lightgbm.py

```diff
@@ -195,19 +195,26 @@ def predict(self, dataframe):
 
 
 @experimental
-def autolog():
+def autolog(normalize=False, max_num_features=None):
     """
     Enables automatic logging from LightGBM to MLflow. Logs the following.
 
     - parameters specified in `lightgbm.train`_.
     - metrics on each iteration (if ``valid_sets`` specified).
     - metrics at the best iteration (if ``early_stopping_rounds`` specified).
-    - feature importance (both "split" and "gain").
+    - feature importance (both "split" and "gain") as JSON files and plots.
     - trained model.
 
+    :param normalize: If ``True``, divide each value in feature importance
```

Comment:
One question around these parameters (I'm curious since I'm not a lightgbm user myself) - how commonly do you imagine users would want to configure these? If it's rare, or if the defaults would work reasonably well, we could just remove these parameters for now & wait for user feedback (& you might even be able to provide this feedback yourself if you've been using lightgbm) that suggests we should include them, as it's good to default to keeping APIs simple where possible.

Reply:
Agreed, will remove them for now.